tests/integration/utils.py

```diff
@@ -0,0 +1,12 @@
+import yaml
```

Comment:
You'll need to add PyYAML to test-requirements.txt to fix the current CI failure.